lib/ansible/module_utils/basic.py

```diff
@@ -1599,7 +1601,7 @@ def _log_invocation(self):
                 if not isinstance(param_val, basestring):
                     param_val = str(param_val)
                 elif isinstance(param_val, unicode):
-                    param_val = param_val.encode('utf-8')
+                    param_val = str(param_val.encode('utf-8'))
```

Comment:
Do not seems to be present. But yeah, because this is likely doing something wrong (as I found out on others part of the code)